index.html

```diff
@@ -38,6 +38,7 @@ <h2 id="language">Language-specific</h2>
       <li><a href="http://www.meetup.com/London-JavaScript-Community/">London JS</a> - meetups and community for JavaScript developers that are interested to learn more about JavaScript, share development experiences or techniques.</li>
       <li><a href="http://www.meetup.com/Londonjavacommunity/">London Java Community</a> - an official Java User Group for developers based in London, with talks, workshops and hackathons.</li>
       <li><a href="https://skillsmatter.com/groups/6-london-dot-net-user-group">London .NET User Group</a> - monthly meetings with talks and regular 'Open Mic' nights to discuss best practice and future directions of the .NET framework.</li>
+      <li><a href="http://london.pm.org/">London Perl Months</a> - monthly social meetups and frequent technical talks for those interested in Perl.</li>
```

Comment:
Thanks for submitting! Just to check, you mentioned the name as London Perl Mongers, but here it says London Perl Months - is that intended?